rclc_lifecycle/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 3.5)
+project(rclc_lifecycle)
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(lifecycle_msgs REQUIRED)
+find_package(rcl_lifecycle REQUIRED)
+find_package(rclc REQUIRED)
+
+# create library
+add_library(${PROJECT_NAME}
+  src/rclc_lifecycle/rclc_lifecycle.c
+)
+if("${rcl_VERSION}" VERSION_LESS "1.0.0")
+  target_sources(${PROJECT_NAME} PRIVATE src/rclc/rcl_wait_set_is_valid_backport.c)
```

Comment:
Is this needed here? The rclc_lifecycle package is not using wait_set_is_valid directly, or? If this is needed to build against rclc, we should think of a better mechanism in the rclc package.

Reply:
It was not required, thanks for the hint!